paths.tex

```diff
@@ -77,9 +77,9 @@ \section{Preliminary Definitions}
 directed temporal networks. To go from undirected to directed, for every edge
 $(e_l,e_r) \in E^2$, add edges $e_l \in E$ and $e_r \in E$ to the directed
 network. To go from directed to undirected, if for every $u,v$ pair in the
-directed network, the number of edges of the form $(u,v,t_0,t_f,w)$ is the same
-as the number of edges of the form $(v,u,t_0,t_f,w)$, then pair up these edges;
-otherwise there is no transformation.
+directed network, the number of edges of the form $e = (u,v)$ is the same
```

Comment:
I'm not partial to this section. Its not important to anything we want to do. We'll more usually just use the underlying graph when we want an undirected from a directed. It was intended to give intuition about the old definition we had. Now we dont need it.